user_docs/en/userGuide.t2t

```diff
@@ -1220,6 +1220,19 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
+==== Report indents as ====
+When Report Line Indentation is enabled, the Report indents as combo box has three options.
+
+- Tones
+- Speech
+- Both Speech and Tones
+
+
+If speech is selected, when the level of indented text changes, NVDA will say something like "twelve spaces" or "four tabs."
+
+If Tones is selected, when the level of indented text changes, tones indicate the amount of change in indent.
+The tone will increase in pitch every space, and for a tab, it will increase in pitch the equivalent of 4 spaces.

```

Comment:
I'm wondering whether these descriptions of speech and tones should be moved up into the list. It seems redundant to have the list and then the descriptions in prose below it.